client/post-editor/editor-discussion/index.jsx

```diff
@@ -136,7 +137,37 @@ export class EditorDiscussion extends React.Component {
 						disabled={ ! this.props.post }
 						onChange={ this.onChange }
 					/>
-					<span>{ this.props.translate( 'Allow Pingbacks & Trackbacks' ) }</span>
+					<span>
+						{ this.props.translate( 'Allow Pingbacks & Trackbacks' ) }
+						<InfoPopover
+							position="top right"
+							className="editor-discussion__status-info"
+							gaEventCategory="Editor"
+							popoverName="PingStatus"
+						>
+							{ this.props.translate(
+								'{{pingbacksLink}}Pingbacks{{/pingbacksLink}} and {{trackbacksLink}}trackbacks{{/trackbacksLink}} are automated comments you will receive when others create links to your post elsewhere.',
+								{
+									components: {
+										pingbacksLink: (
+											<ExternalLink
+												href="https://support.wordpress.com/comments/pingbacks/"
+												target="_blank"
+												icon="true"
+											/>
+										),
+										trackbacksLink: (
+											<ExternalLink
+												href="https://support.wordpress.com/comments/trackbacks/"
+												target="_blank"
+												icon="true"
```

Comment:
Excellent, the images in the original post have been updated now. ✅ 

I am leaving this conversation open, in case any design members want to chime in on the same thread.